src/test/java/de/tum/in/www1/artemis/authentication/UserJWTIntegrationTest.java

```diff
@@ -0,0 +1,76 @@
+package de.tum.in.www1.artemis.authentication;
+
+import static de.tum.in.www1.artemis.util.ModelFactory.USER_PASSWORD;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.test.context.support.WithAnonymousUser;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+
+import de.tum.in.www1.artemis.AbstractSpringIntegrationTest;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.repository.*;
+import de.tum.in.www1.artemis.security.jwt.TokenProvider;
+import de.tum.in.www1.artemis.service.UserService;
+import de.tum.in.www1.artemis.util.DatabaseUtilService;
+import de.tum.in.www1.artemis.util.ModelFactory;
+import de.tum.in.www1.artemis.util.RequestUtilService;
+import de.tum.in.www1.artemis.web.rest.UserJWTController;
+import de.tum.in.www1.artemis.web.rest.vm.LoginVM;
+
+@ActiveProfiles("artemis")
+@TestPropertySource(properties = "artemis.user-management.use-external=false")
+public class UserJWTIntegrationTest extends AbstractSpringIntegrationTest {
```

Comment:
can you move this into `InternalAuthenticationIntegrationTest` so that a we do not have 2 tests with different profiles?

This can significantly slow down test execution

Reply:
Good point, I moved it into `InternalAuthenticationIntegrationTest`.